python/ray/experimental/state/state_cli.py

```diff
@@ -173,7 +173,15 @@ def output_with_format(
     if format == AvailableFormat.DEFAULT:
         return get_table_output(state_data, schema)
     if format == AvailableFormat.YAML:
-        return yaml.dump(state_data, indent=4, explicit_start=True)
+        augmented_state_data = [
+            {
+                ("task_id: " + state["task_id"])
```

Comment:
I guess we might want to use the extra task id as the key for this dict?

Reply:
hmm, I am not sure what you mean, are you saying we should pull out the state[task_id] into a separate local variable to re-use elsewhere?